selfdrive/car/body/fingerprints.py

```diff
@@ -0,0 +1,36 @@
+from cereal import car
+from selfdrive.car.body.values import CAR
+from selfdrive.car.fw_query_definitions import FwQueryConfig, Request, StdQueries
+
+Ecu = car.CarParams.Ecu
+
+FINGERPRINTS = {
+  CAR.BODY: [{
+    513: 8, 516: 8, 514: 3, 515: 4,
+  }],
+}
+
+FW_QUERY_CONFIG = FwQueryConfig(
```

Comment:
think this should still live in values.py. the idea is that these files eventually become largely auto-generated by an offline job pulling in fuzzy fingerprints

Reply:
As far as I know, only Hyundai will have the circular import problem (since it has a fuzzy fingerprinting function that imports from fingerprints.py). We can just do the importing at run time instead of import time (in a cached function) with `get_interface_attr`. Not sure how clean that is though

Reply:
Though I eventually wanted to make the refactor to put the FW versions inside the config, and we could combine a bunch of logic by having the configs manage everything, which this makes harder.